io_scene_xray/details/operators.py

```diff
@@ -61,7 +61,7 @@ def execute(self, context):
             self.report({'ERROR'}, 'No files selected')
             return {'CANCELLED'}
 
-        import_context = fmt_object_imp.ImportContext(
+        import_context = object_.import_.ImportContext(
```

Comment:
_minor:_ `_` suffixes look a bit ugly. Did you consider using `obj`, `imp`, and `exp` abbreviation instead of `object_`, `import_`, and `export` words? I just wonder if abbreviations would look better than suffixes.

Reply:
Хорошо, переименую. И длина строк уменьшиться.

Reply:
Кстати, в PEP8 рекомендуют использовать в конце нижнее подчёркивание, если имя конфликтует с ключевыми словами и со стандартной библиотекой. Так что этот способ именования не противоречит PEP8. Но я всё же переименую.